app/models/invenio_rdm_record_converter.rb

```diff
@@ -90,6 +90,8 @@ def invenio_metadata(gf)
       "additional_titles": gf.title.last(gf.title.size-1).map{ |title| {"title": title, "type": "alternative_title", "lang": "eng"} },
       "description": gf.description.first,
       "additional_descriptions": gf.description.last(gf.description.size-1).map{ |add_desc| {"description": add_desc, "type": "other", "lang": "eng"} },
+      "publisher": gf.publisher.shift,
+      "publication_date": "#{gf.date_uploaded.month}-#{gf.date_uploaded.day}-#{gf.date_uploaded.year}",
```

Comment:
For invenioRDM, should this be YEAR-MONTH-DAY?

Reply:
You're right. I think I made this change before the follow up with just the `date_created` field. I'll fix this up before merging.